project/FrgaalJavaCompiler.scala

```diff
@@ -213,9 +230,15 @@ object FrgaalJavaCompiler {
     withArgumentFile(allArguments) { argsFile =>
       // Need to disable standard compiler tools that come with used jdk and replace them
       // with the ones provided with Frgaal.
-      val forkArgs = (jArgs ++ Seq(
-        "--limit-modules",
-        "java.base,jdk.zipfs,jdk.internal.vm.compiler.management",
+      val limitModulesArgs = if (noLimitModules) {
+        Seq()
+      } else {
+        Seq(
+          "--limit-modules",
+          "java.base,jdk.zipfs,jdk.internal.vm.compiler.management",
```

Comment:
I think I got the processor working with simple:
```java
        Seq(
          "--limit-modules",
          "java.base,jdk.zipfs,jdk.internal.vm.compiler.management,org.graalvm.locator",
        )
```
e.g. looks like adding `org.graalvm.locator` is the change that makes the SDK work.

Reply:
This does not work anymore. See my previous comment https://github.com/enso-org/enso/pull/7101#discussion_r1281601789.